src/hotspot/share/ci/ciInstanceKlass.cpp

```diff
@@ -391,17 +391,23 @@ bool ciInstanceKlass::contains_field_offset(int offset) {
   return get_instanceKlass()->contains_field_offset(offset);
 }
 
+// ------------------------------------------------------------------
+// ciInstanceKlass::get_non_static_field_by_offset
```

Comment:
I think you can remove this - we used to add it in the early days but not anymore.

```suggestion
```